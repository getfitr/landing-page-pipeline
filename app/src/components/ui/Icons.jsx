import React from 'react';
import { CgClose } from 'react-icons/cg';
import { FiMenu } from 'react-icons/fi';

const CloseIcon = () => <CgClose />;
const MenuIcon = () => <FiMenu />;

export {
  CloseIcon,
  MenuIcon,
}