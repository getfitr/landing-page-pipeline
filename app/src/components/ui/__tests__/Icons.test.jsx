import React from 'react';
import renderer from 'react-test-renderer';
import { CloseIcon, MenuIcon } from '../Icons';

test('Display close icon', () => {
  const logo = renderer.create(<CloseIcon />)
  const tree = logo.toJSON()
  expect(tree).toMatchSnapshot();
});

test('Display menu icon', () => {
  const logo = renderer.create(<MenuIcon />)
  const tree = logo.toJSON()
  expect(tree).toMatchSnapshot();
});
