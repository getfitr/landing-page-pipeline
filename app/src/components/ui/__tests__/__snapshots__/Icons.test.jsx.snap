// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display close icon 1`] = `
<svg
  fill="none"
  height="1em"
  stroke="currentColor"
  strokeWidth="0"
  style={
    Object {
      "color": undefined,
    }
  }
  viewBox="0 0 24 24"
  width="1em"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M6.2253 4.81108C5.83477 4.42056 5.20161 4.42056 4.81108 4.81108C4.42056 5.20161 4.42056 5.83477 4.81108 6.2253L10.5858 12L4.81114 17.7747C4.42062 18.1652 4.42062 18.7984 4.81114 19.1889C5.20167 19.5794 5.83483 19.5794 6.22535 19.1889L12 13.4142L17.7747 19.1889C18.1652 19.5794 18.7984 19.5794 19.1889 19.1889C19.5794 18.7984 19.5794 18.1652 19.1889 17.7747L13.4142 12L19.189 6.2253C19.5795 5.83477 19.5795 5.20161 19.189 4.81108C18.7985 4.42056 18.1653 4.42056 17.7748 4.81108L12 10.5858L6.2253 4.81108Z"
    fill="currentColor"
  />
</svg>
`;

exports[`Display menu icon 1`] = `
<svg
  fill="none"
  height="1em"
  stroke="currentColor"
  strokeLinecap="round"
  strokeLinejoin="round"
  strokeWidth="2"
  style={
    Object {
      "color": undefined,
    }
  }
  viewBox="0 0 24 24"
  width="1em"
  xmlns="http://www.w3.org/2000/svg"
>
  <line
    x1="3"
    x2="21"
    y1="12"
    y2="12"
  />
  <line
    x1="3"
    x2="21"
    y1="6"
    y2="6"
  />
  <line
    x1="3"
    x2="21"
    y1="18"
    y2="18"
  />
</svg>
`;
